pymc_marketing/mmm/mmm.py

```diff
@@ -92,6 +92,14 @@ def __init__(
         ),
         model_config: dict | None = Field(None, description="Model configuration."),
         sampler_config: dict | None = Field(None, description="Sampler configuration."),
+        allow_negative_channels: Annotated[
```

Comment:
What would be the effect of this new flag on the model? Change the prior? What about simply passing the data multiplied by -1 (for the desired channels) to the original model?

Reply:
No change really! No change to prior etc- it's just that currently to be able to pass in data that's negative / you've multiplied by -1 you have to set `validate_data=False` in `MMM()` and I'd prefer to be able to retain the validation while still passing in the negative values.

Or perhaps I'm misunderstanding something haha. A very lightweight PR so far so happy to close if I'm misunderstanding something ☺️

Reply:
ok! I get the idea :) 

I generally try to avoid flags as this hints at design flaws (from our side)

An alternative would be to remove:

https://github.com/pymc-labs/pymc-marketing/blob/main/pymc_marketing/mmm/validating.py#L118

Or make it optional in a better way .... Ideally with Pydantic ... 

Let me think about it (we can keep this PR open) 🙏 

@wd60622 any thoughts or ideas?

Reply:
Ok! @louismagowan, would you mind removing the ValueError and adding a Warning instead?

Reply:
@juanitorduz should be good now I think! ☺️